invenio_oaiharvester/api.py

```diff
@@ -0,0 +1,107 @@
+# -*- coding: utf-8 -*-
+#
+# This file is part of Invenio.
+# Copyright (C) 2015 CERN.
+#
+# Invenio is free software; you can redistribute it and/or
+# modify it under the terms of the GNU General Public License as
+# published by the Free Software Foundation; either version 2 of the
+# License, or (at your option) any later version.
+#
+# Invenio is distributed in the hope that it will be useful, but
+# WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+# General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with Invenio; if not, write to the Free Software Foundation, Inc.,
+# 59 Temple Place, Suite 330, Boston, MA 02111-1307, USA.
+from sickle import Sickle
+from invenio_oaiharvester.errors import (NameOrUrlMissing, WrongDateCombination,
+                                         NAME_OR_URL_ERROR_MSG, WRONG_DATES_ERROR_MSG)
+from invenio_oaiharvester.models import OaiHARVEST
```

Comment:
We are moving to use relative imports within the same module:

`from .models import OaiHARVEST` Same for `.errors`.